Update agent setup instructions (#94)
neo4j/docs-ops-manager: modules/ROOT/pages/addition/agent-installation/agent-config-file.adoc

= Agent configuration file

> Agent configuration file is used by a self-registering agent to persist it's initial configuration which it requires in restart scenarios.

Reviewer: ```suggestion
The agent configuration file is used by a self-registering agent to persist its initial configuration, which is required for restarts.
```

Agent looks for a file named *`nom-agent-config.yaml`* under the following directories in order:
. path set by `CONFIG_AGENT_CONFIG_PATH` environment variable
. `conf` directory under path set by `NEO4J_HOME` environment variable
. path set by `NEO4J_CONF` environment variable
. `.nom` directory under user home directory by default

[WARNING]
====

> Agent reports an error while running in self-register mode such as `| ERROR | unable to load config from file error="open /home/.nom/nom-agent-config.yaml: no such file or directory"`. It is not an issue with agent operations but default behaviour to support restarts of agents which needs to llok for its initial configuration to connect to NOM server.

Reviewer: ```suggestion
Agent initially reports an error while running in self-register mode such as `| ERROR | unable to load config from file error="open /home/.nom/nom-agent-config.yaml: no such file or directory"`. This is expected behavior, as the file will be created automatically.
```

Reviewer: I think we can simplify this a bit
